### PR TITLE
dotnet-add-{package, reference}: add page

### PR DESCRIPTION
**Pull request by blueskyson**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** since .NET 3.1

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/dotnet-add-reference.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# dotnet add reference
+
+> Adds .NET project-to-project references.
```
Comment (acuteenvy) [inline pages/common/dotnet-add-reference.md:3]:
```suggestion
> Add .NET project-to-project references.
```

File: pages/common/dotnet-add-package.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# dotnet add package
+
+> Add or update .NET package reference in a project file.
```
Comment (acuteenvy) [inline pages/common/dotnet-add-package.md:3]:
```suggestion
> Add or update a .NET package reference in a project file.
```

File: pages/common/dotnet-add-package.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Add a package using a specific NuGet source:
+
+`dotnet add package {{package}} --source {{https://api.nuget.org/v3/index.json}}`
+
+- Add a package only when targeting a specific framework:
+
+`dotnet add package {{package}} --framework {{net7.0}}`
+
+- Add and specify the directory where to restore packages. (In `~/.nuget/packages` by default):
```
Comment (acuteenvy) [inline pages/common/dotnet-add-package.md:26]:
```suggestion
- Add and specify the directory where to restore packages (`~/.nuget/packages` by default):
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**